t/024_boolean_header_fields.t

```diff
@@ -2,6 +2,10 @@ use Test::More;
 use strict;
 use warnings;
 
+plan skip_all => "this test has not been converted as there's public admin on the new server";
```

Comment:
Why not use the server specified by the env var MQHOST? The Net::AMQP::RabbitMQ tests use this one and you can set it for the travis tests for whatever server you want to use there.